css/04-content.css

```diff
@@ -149,3 +149,188 @@ If still here by 8/10/2019, remove this code.
 .give-currency-symbol {
 	font-weight: bold;
 }
+
+
+/****************************************/
+/* Main Menu / Branding Container */
+/****************************************/
+
+/* set branding container to the top and add color*/
+.featured-image .site-branding-container,
+.site-header:not(.featured-image) {
+	background-color:  #fff;
+	z-index: 10;
+}
+
+/* recenter featured image */
+.site-header.featured-image .site-featured-image .post-thumbnail img {
+	top: 5rem;
+}
+
+/*remove default button shadows */
+.main-navigation .main-menu > li.menu-item-has-children .c4aa-default-button-wrapper {
+	filter: none;
+}
+
+/* style logo hover */
+.site-logo .custom-logo-link:hover, 
+.site-logo .custom-logo-link:active, 
+.site-logo .custom-logo-link:focus,
+.site-header.featured-image .custom-logo-link:hover,
+.site-header.featured-image .custom-logo-link:active,
+.site-header.featured-image .custom-logo-link:focus {
+	box-shadow: none;
+	opacity: 0.6;
+}
+
+/* style text */
+.site-title {
+	font-family: 'roboto', Georgia, serif;
+	font-weight: bold;
+	margin: 0;
+	text-transform: uppercase;
+	margin-top: 10px;
+	
+}
+
+.site-title span::after {
+	content: '\25CF';
+	font-size: .7rem;
+	margin-bottom: .5rem;
+	vertical-align: top;
+	color: var( --color-brandBlack );
+	width: 1rem;
+}
+
+.site-title a:link, .site-title a:visited,
+.site-header.featured-image .site-title a,
+.site-header.featured-image .site-title a:hover,
+.site-header.featured-image .main-navigation a, 
+.site-header.featured-image .main-navigation a + svg, 
+.site-header.featured-image .social-navigation a, 
+.site-header.featured-image .site-featured-image a,
+.main-navigation .main-menu > li > a, 
+.main-navigation .main-menu > li .svg-icon  {
+	color:  var( --color-brandBlack );
+
+}
+
+.site-header.featured-image .main-navigation a:hover, 
+.site-header.featured-image .main-navigation a:active, 
+.site-header.featured-image .main-navigation a + svg:hover, 
+.site-header.featured-image .main-navigation a + svg:active, 
+.site-header.featured-image .social-navigation a:hover, 
+.site-header.featured-image .social-navigation a:active, 
+.site-header.featured-image .site-featured-image a:hover 
+.site-header.featured-image .site-featured-image a:active {
+	color:  var( --color-brandRed );
+	opacity: 1;
+}
+
+.main-menu li {
+	font-size: 1rem;
+}
+
+/*  position elements 
+
+	containers: 
+		.site-branding
+		.featured-image .site-branding-container,
+		.site-header
+
+	logo: 
+		.site-logo
+		.site-logo img
+
+	title:
+		.site-title
+
+	nav links:
+			#site-navigation
+			#menu-2019menu
+			#site-navigation ul li 
+
+
+*/
+
+.site-branding::after {
+	clear: both;
```

Comment:
Why did you decide to use float layout here? Can we use flexbox or grid instead?

Reply:
I originally did this with flexbox, but for some reason it caused the ellipses icon (from the mobile view) to show even if all the menu items are showing. I tried to dig into why that was happening but it seems to be some sort of PHP or JS code form the twenty-nineteen theme. I honestly am not sure what could be causing that.

Reply:
@laras126 So I figured out how to dequeue the script that causes the ellipsis to show up, and submitted a commit for that. This hopefully will allow us to use flexbox o grid here, and establish our own mobile menu styles. But I am working on it.

Reply:
@laras126 So I figured out how to dequeue the script that causes the ellipsis to show up and cut the menu off, and I set a flex style instead of float.  I think this should be ready to commit.
@slambert We can set up a hamburger menu later, as well.